tools/jvm-options-parser/build.gradle

```diff
@@ -31,8 +31,8 @@ buildscript {
   }
 }
 
-project.sourceCompatibility = JavaVersion.VERSION_1_8
-project.targetCompatibility = JavaVersion.VERSION_1_8
+project.sourceCompatibility = JavaVersion.VERSION_11
+project.targetCompatibility = JavaVersion.VERSION_11
```

Comment:
**Note for reviewer**

Running Gradle 8.5 with JDK 21 with
```sh
./gradlew :jvm-options-parser:compileJava
```
trigger the error:
```
> Task :jvm-options-parser:compileJava FAILED
warning: [options] source value 8 is obsolete and will be removed in a future release
warning: [options] target value 8 is obsolete and will be removed in a future release
warning: [options] To suppress warnings about obsolete options, use -Xlint:-options.
error: warnings found and -Werror specified

FAILURE: Build failed with an exception.
```

Java 8 is obsolete and its support will be dropped